src/widget/widget.cpp

```diff
@@ -470,7 +470,9 @@ void Widget::onStatusSet(Status status)
 
 void Widget::setWindowTitle(const QString& title)
 {
-    QMainWindow::setWindowTitle("qTox - " + title);
+    QString tmp = title;
+    /// <[^>]*> Regexp to remove HTML tags, in case someone used them in title
+    QMainWindow::setWindowTitle("qTox - " + tmp.remove(QRegExp("<[^>]*>")));
```

Comment:
Please be careful not to send commits unrelated to the PR's title and description, ever. I'll just cherry-pick the last commit this time.